cmd/translate/root.go

```diff
@@ -0,0 +1,59 @@
+package translate
```

Comment:
root.go should be at the root for cmd, and translate.go is root for translate 
https://github.com/translate-agent/translate/pull/7#discussion_r1094332358

Reply:
I also thought like that, but I had a call with Janis about this, he said that root.go should be in translate directory

Reply:
@janishorsts could you explain why?  🤔

Reply:
We'll have multiple CLI commands.

Reply:
if understand this file should be called `main.go` e.g. `cmd/translate/main.go.`

Files `cmd/translate/translate` and `main.go` should be deleted